appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -1738,21 +1738,19 @@ public void Sizing(String value) {
    */
 
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_BOOLEAN,
-    defaultValue = "False")
+    defaultValue = "True")
```

Comment:
You should also include `alwaysSend = true` here. Typically, we only send a value if it differs from the default. However, since we also want consistent behavior depending on the version of the companion (older versions will have the old behavior as default), alwaysSend tells the system to send regardless of whether we match the default so that the actual desired behavior is asserted.

Reply:
Changed this in next commit